rcl_lifecycle/src/rcl_lifecycle.c

```diff
@@ -275,19 +254,15 @@ rcl_lifecycle_state_machine_fini(
 rcl_ret_t
 rcl_lifecycle_state_machine_is_initialized(const rcl_lifecycle_state_machine_t * state_machine)
 {
-  if (!state_machine->com_interface.srv_get_state.impl) {
-    RCL_SET_ERROR_MSG("get_state service is null");
-    return RCL_RET_ERROR;
-  }
-  if (!state_machine->com_interface.srv_change_state.impl) {
-    RCL_SET_ERROR_MSG("change_state service is null");
-    return RCL_RET_ERROR;
-  }
-  if (rcl_lifecycle_transition_map_is_initialized(&state_machine->transition_map) != RCL_RET_OK) {
-    RCL_SET_ERROR_MSG("transition map is null");
```

Comment:
missing to set error message. I think that it would be better to be independent bug fix only.

Reply:
Got it! I will revert this part.

Reply:
@fujitatomoya 
Any tips?

Reply:
sorry my memory is little hazy, but what i meant is replace RCL_SET_ERROR_MSG into RCL_CHECK_FOR_NULL_WITH_MSG, also related to https://github.com/ros2/rcl/pull/742/files#r471208047

Reply:
Now it came back! previously "transition map is null" error message is deleted, i was thinking that it would be better to keep the error message to be set.

Reply:
I have update the code to revert it back.